cli/src/commands/status.rs

```diff
@@ -67,6 +69,12 @@ pub(crate) fn cmd_status(
             writeln!(formatter, "The working copy is clean")?;
         } else {
             writeln!(formatter, "Working copy changes:")?;
+            let mut copy_records = CopyRecords::default();
+            for parent in wc_commit.parents() {
```

Comment:
This could probably be `wc_commit.parent_ids()`, unless the intent was to cache the commit?